addon/utils.js

```diff
@@ -22,6 +22,14 @@ export const ensureList = (item) => item == null
 
 export const getFirstKey = (obj) => Object.keys(obj)[0];
 
+export const getUnionField = (type, fieldName) => {
+  for (const t of type._types) {
+    if (t._fields[fieldName]) {
```

Comment:
This might be an edge case, but would there be an issue if multiple types had the same `fieldName` but the fields were different types?

Reply:
That would be an issue but addressing it seems like a much larger piece of work that can potentially come later.

Reply:
Yes, this would be an issue. I feel the appropriate thing would be to add full UnionType functionality at that point, where this is a simple fix that allows us to add UnionTypes and use mapping to accomplish it